src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/uninstrumented-entities-service-maps.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Uninstrumented entities in service maps'
+tags:
+  - Understand dependencies
+  - Understand system dependencies
+  - Uninstrumented entities
+  - Service maps
+metaDescription: Uninstrumented entities are created by New Relic and can be easily instrumented by the customer
+---
+import ServiceMapUninstrumentedEntity from 'images/mapsSolutions_screenshot-crop_uninstrumented-entity.png'
+
+New Relic can detect gaps on your instrumentation by analyzing your telemetry.These gaps can be databases, apps, or basically any other type of entity. For example, one application could be storing information in a non-instrumented Amazon RDS database.
```

Comment:
Please let's use `uninstrumented` instead of `non-instrumented` everywhere, it is something we discussed in the past and decided to stick w/ the `uninstrumented` term.

```suggestion
New Relic can detect gaps on your instrumentation by analyzing your telemetry. These gaps can be databases, apps, or basically any other type of entity. For example, one application could be storing information in a uninstrumented Amazon RDS database.
```